src/chart/tree/TreeView.js

```diff
@@ -379,9 +379,8 @@ function updateNode(data, dataIndex, symbolEl, group, seriesModel, seriesScope)
         // Fix #12279.
         // if the type of symbol is image,
         // update symbol's data immediately but not wait until the next update rendering.
-        var symbolType = symbolEl._symbolType;
-        symbolType.indexOf('image://') === 0 
-            && symbolEl.updateData(data, dataIndex, seriesScope);
+        var symbolType = symbolEl.getSymbolPath().type;
+        symbolType === 'image' && symbolEl.updateData(data, dataIndex, seriesScope);
```

Comment:
I'm thinking which scheme will be better here?

Before
```js
var symbolType = symbolEl._symbolType; // easy to get, but it is private, can I access it here?
symbolType.indexOf('image://') === 0 && symbolEl.updateData(data, dataIndex, seriesScope);
```
Now
```js
var symbolType = symbolEl.getSymbolPath().type;
symbolType === 'image' && symbolEl.updateData(data, dataIndex, seriesScope);
```

Reply:
@plainheart 
`updateData` has been called in the constructor of `Symbol`, I think it should better not be called again here.
The actual reason of this issue seams to be in: 
https://github.com/apache/incubator-echarts/blob/4.8.0/src/chart/helper/Symbol.js#L241
The reset of "image" symbol should not set `opacity` to `null`.
`null` is an invalid value for `opacity`.
I think here the opacity should be reset as `1`. 
And that can fix this issue.

Reply:
Yes, you are totally right. I've made a new commit for this. https://github.com/apache/incubator-echarts/pull/12367/commits/48d54d8b604418b97b74e4da688c6a9699084d61